design/ceph/object/ceph-rgw-support-multisite-for-external-cluster.md

```diff
@@ -0,0 +1,31 @@
+---
+# Support Realm/Zone/Zonegroup for external RGW server
+target-version: release-X.X
+---
+
+# Feature Name
+ Support Realm/Zone/Zonegroup for external RGW server
+
+## Summary
+Rook supports handling for external RGW servers by assuming it is part of `default` zone/zonegroup. To handle consistently it is better handle this info, otherwise the radosgw apis may fail for such RGW servers.
+
+### Goals
+If external RGW server is configured with zone/zonegroup/realm, allow equivalent CR creation in Rook.
```

Comment:
For an external RGW, rook doesn't create any RGW pod. So why do we need to create these CRs in Rook for the external RGW case?

Reply:
My alternate approach to defining these directly is on the `ObjectStoreSpec`. I am still guessing the `realm` k8s secret still needs to be created since considering the situation in which one cluster is external and the secondary one is internal